specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/publicIpAddress.json

```diff
@@ -444,6 +444,18 @@
           "readOnly": true,
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the public IP address resource."
+        },
+        "servicePublicIPAddress": {
+          "$ref": "#/definitions/PublicIPAddress",
+          "description": "The service public IP address of the public IP address resource."
+        },
+        "natGateway": {
+          "$ref": "./network.json#/definitions/NatGateway",
```

Comment:
This reference model doesn't exist.

Reply:
@raych1  Could you guide how can I mention the reference to NatGateway model here? what should be the correct syntax?

Reply:
The syntax is correct and the issue is NatGateway doesn't exist in network.json under 2020-08-01 folder.